tests/scene/test_text_edit.h

```diff
@@ -3298,6 +3298,11 @@ TEST_CASE("[SceneTree][TextEdit] setter getters") {
 	TextEdit *text_edit = memnew(TextEdit);
 	SceneTree::get_singleton()->get_root()->add_child(text_edit);
 
+	SUBCASE("[TextEdit] get scroll bar") {
+		CHECK(text_edit->get_h_scroll_bar() != nullptr);
+		CHECK(text_edit->get_v_scroll_bar() != nullptr);
+	}
```

Comment:
Is this test case really useful?

Reply:
Fwiw, it also tests that the functions exist and are defined correctly. 

Related, how strict is this project with unit test code coverage? 

I'm happy to remove this test if you prefer!

Reply:
> Fwiw, it also tests that the functions exist and are defined correctly.

We don't do API tests, so this is indeed unnecessary. If these functions get removed, `bind_methods()` will fail at compile time.

Reply:
Noted! I can remove.